internal/common/common.go

```diff
@@ -18,6 +18,12 @@ const HostCACertPath = "/etc/assisted-service/service-ca-cert.crt"
 
 const consoleUrlPrefix = "https://console-openshift-console.apps"
 
+const MirrorRegistriesCertificateFile = "mirror_ca.pem"
+const MirrorRegistriesCertificatePath = "/etc/pki/ca-trust/extracted/pem/" + MirrorRegistriesCertificateFile
+const MirrorRegistriesConfigDir = "/etc/containers"
+const MirrorRegistriesConfigFile = "registries.conf"
+const MirrorRegistriesConfigPath = MirrorRegistriesConfigDir + "/" + MirrorRegistriesConfigFile
```

Comment:
https://golang.org/doc/effective_go#constants and https://golang.org/doc/effective_go#commentary make me think that we should be doing something like:

```suggestion
// important keys for handling a mirror registry
const (
	MirrorRegistriesCertificateFile = "mirror_ca.pem"
	MirrorRegistriesCertificatePath = "/etc/pki/ca-trust/extracted/pem/" + MirrorRegistriesCertificateFile
	MirrorRegistriesConfigDir = "/etc/containers"
	MirrorRegistriesConfigFile = "registries.conf"
	MirrorRegistriesConfigPath = MirrorRegistriesConfigDir + "/" + MirrorRegistriesConfigFile
)
```

Reply:
fixed